mne/viz/topo.py

```diff
@@ -465,6 +465,7 @@ def _erfimage_imshow_unified(bn, ch_idx, tmin, tmax, vmin, vmax, ylim=None,
 
 def _plot_evoked_topo(evoked, layout=None, layout_scale=0.945, color=None,
                       border='none', ylim=None, scalings=None, title=None,
+                      legend=True, legend_loc=0, legend_fontsize=10,
```

Comment:
these options are added but not exposed as it's just a private function. any reason for this?